include/pros/rtos.hpp

```diff
@@ -820,6 +820,63 @@ struct Clock {
 	static time_point now();
 };
 
+template <class T>
+class Queue {
+	queue_t queue;
+
+	public:
```

Comment:
```suggestion
	public:
	static_assert(std::is_trivially_copyable_v<T>);
```

Reply:
I'm not going to use static assert, it does not work well with intellisense. I'll use concept.